update_service/index.adoc

```diff
@@ -0,0 +1,48 @@
+:context: update-service
+[id="update-service"]
+= Understanding the {product-title} Update Service
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+// The following include statements pull in the module files that comprise
+// the assembly. Include any combination of concept, procedure, or reference
+// modules required to cover the user story. You can also include other
+// assemblies.
+
+include::modules/update-service-overview.adoc[leveloffset=+1]
+
+[id="update-service-install"]
+== Installing the {product-title} Update Service
+include::modules/common-attributes.adoc[]
+
+// You can install the {product-title} Update Service by deploying the Update Service operator, using the {product-title} xref FIXME ../../update_service/install/web-console.adoc[web console] or xref FIXME ../../update_service/install/cli.adoc[CLI].
+
+// The following include statements pull in the module files that comprise
+// the assembly. Include any combination of concept, procedure, or reference
+// modules required to cover the user story. You can also include other
+// assemblies.
+
+include::modules/update-service-install-web-console.adoc[leveloffset=+2]
+
+include::modules/update-service-install-cli.adoc[leveloffset=+2]
+
+== Additional resources
+
+* For more information on installing Operators, see xref:../operators/admin/olm-adding-operators-to-cluster.adoc#olm-installing-operators-from-operatorhub_olm-adding-operators-to-a-cluster[Installing Operators from the OperatorHub].
+
+include::modules/update-service-graph-data.adoc[leveloffset=+2]
+
+[id="update-service-create-service"]
+== Creating an Update Service
+include::modules/common-attributes.adoc[]
+
+// You can create Update Services using the {product-title} xref FIXME ../../update_service/create-service/web-console.adoc[web console] or xref FIXME ../update_service/create-service/cli.adoc[CLI].
```

Comment:
```suggestion
// You can create Update Services by using the {product-title} xref FIXME ../../update_service/create-service/web-console.adoc[web console] or xref FIXME ../update_service/create-service/cli.adoc[CLI].
```
Please fix these xrefs, too.

Reply:
Picked up the `by` in adf5ca0732 -> 2c77ff9ce2